internal/testonly/docker/cloudbuild.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 Google LLC. All Rights Reserved.
```

Comment:
Is it worth having a chew over build tags vs. an ENV var? Can chat offline if you like?

Reply:
Yeah let's talk about this a bit.